### PR TITLE
improvement: S3C-2675-add-audit-log-fields

### PR DESCRIPTION
**Pull request by anurag4DSB**
This PR adds audit log fields for Cloudserver without adding an extra log line

Extra fields: 
accountDisplayName: account name
iamDisplayName: username

### CODE REVIEW

File: lib/auth/Vault.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ function vaultSignatureCb(err, authInfo, log, callback, streamingV4Params) {
     const info = authInfo.message.body;
     const userInfo = new AuthInfo(info.userInfo);
     const authorizationResults = info.authorizationResults;
+    const auditLog = { accountDisplayName: userInfo.getAccountDisplayName() };
+    const iamDisplayName = userInfo.getIAMdisplayName();
+    if (iamDisplayName) auditLog.IAMdisplayName = iamDisplayName;
```
Comment (ilkescality) [inline lib/auth/Vault.js]: personally prefer curly braces around if statements (at least when it comes to JS), could also make it more consistent as the others seem to have in the same repo

**Review by ilkescality — APPROVED** [review]
LGTM (except seems like bert-e complaining due to using the number of the epic instead of the ticket, might need branch name update)

**Comment by anurag4DSB** [comment]
Thats my bad, had too many tickets open I think :D Will amend as well along with if statements :)

**Comment by anurag4DSB** [comment]
@ilkescality opened new PR with the right ticket: https://github.com/scality/Arsenal/pull/1099
:)

**Pull request closed without merging**